src/main/scala/RemoraApp.scala

```diff
@@ -25,16 +16,6 @@ object RemoraApp extends App with nl.grons.metrics.scala.DefaultInstrumented wit
   metricRegistry.registerAll(new MemoryUsageGaugeSet)
   metricRegistry.registerAll(new ThreadStatesGaugeSet)
 
-  lazy val decider: Supervision.Decider = {
-    case _: IOException | _: ConnectException | _: TimeoutException => Supervision.Restart
-    case NonFatal(err: Throwable) =>
-      actorSystem.log.error(err, "Unhandled Exception in Stream: {}", err.getMessage)
-      Supervision.Stop
-  }
-
-  implicit val materializer = ActorMaterializer(
-    ActorMaterializerSettings(actorSystem).withSupervisionStrategy(decider))(actorSystem)
-
```

Comment:
Is there a reason why you deiced to remove custom supervision strategy?
From the scala doc of `withSupervisionStrategy` method
```
  @deprecated("Use attribute 'ActorAttributes.supervisionStrategy' to change setting value", "2.6.0")
```

Do we need to leave `decider:  Supervision.Decider` and override supervision strategy with the call below?
```
  ActorAttributes.supervisionStrategy(decider)
```

Reply:
To be honest I just saw it was for the ActorMaterializer and deleted all that code.

Will add it back now